src/scribe_data/cli/total.py

```diff
@@ -212,6 +212,12 @@ def get_total_lexemes(language, data_type, doPrint=True):
     else:
         data_type_qid = get_qid_by_input(data_type)
 
+    # Validate language and data type, raise error if invalid
+    try:
+        validate_language_and_data_type(language_qid, data_type_qid)
+    except ValueError as e:
+        print(str(e))
```

Comment:
No need to call string within print :)

Reply:
Or not given the error :) I guess it does make sense that if it's an error then we need to convert it to its string.